index.js

```diff
@@ -41,6 +42,19 @@ function ldnode (argv) {
     // Setting options as local variable
     app.locals.ldp = opts;
 
+    // Setting CORS
+    app.use(cors({
+        methods: [
+            'OPTIONS', 'HEAD', 'GET',
+            'PATCH', 'POST', 'PUT', 'MKCOL', 'DELETE',
+            'COPY', 'MOVE', 'LOCK', 'UNLOCK'
```

Comment:
These methods are not supported yet. I just wanted to align them with gold, so I will just remove them. I am implementing options at the moment.